modules/ServerAPI/src/server/routes/InventoryServiceRoutes.coffee

```diff
@@ -2304,3 +2305,43 @@ formatContainers = (containers, modifiedBy, modifiedDate, callback) ->
 # 	console.log 'locationArrayString '
 # 	console.log locationArrayString
 # 	callback locationArrayString
+
+exports.getContainerInfoFromBatchCode = (req, resp) =>
+
+	requestObject =
+		batchCode: req.body.batchCode
+
+	exports.getContainerInfoFromBatchCodeInternal(requestObject, (json, statusCode) =>
+		resp.statusCode = statusCode
+		resp.json json
+	)
+
+exports.getContainerInfoFromBatchCodeInternal = (requestObject, callback) =>
+
+	queryPayload =
+		{
+			"containerType": "well",
+			"containerKind": "default",
+			"stateType": "status",
+			"stateKind": "content",
+			"valueType": "codeValue",
+			"valueKind": "batch code",
+			"value": requestObject.batchCode,
+			"maxResults": 600,
```

Comment:
@brianbolt - where are configs pulled in from in base acas, same place as custom?